src/site/_includes/partials/post.njk

```diff
@@ -14,6 +14,25 @@
   {# Make sure Actions are first in the tab order. #}
   <web-actions actions="share|subscribe" authors="{{ authors | expandContributors | join('|') }}"></web-actions>
 
+  {% set gitlocalizeUrl =  page.inputPath | gitlocalizeLink(lang) %}
+  {% if translation %}
+    {% Banner 'neutral' %}
+      {% if translation === 'none' %}
+        This page is not translated yet.
+      {% elif translation === 'machine' %}
+        This page is automatically translated using the Google Translation
+        API.
+      {% elif translation === 'manual' %}
+        This article was translated from
+        [the original post](/en{{ page.url | stripLanguage}}).
+      {% endif %}
+      {% if gitlocalizeUrl %}
+        Consider improving it by contributing your translation
+        [here]({{ gitlocalizeUrl }}).
```

Comment:
```suggestion
        Consider improving it by [contributing your translation
        here]({{ gitlocalizeUrl }}).
```

Reply:
Small request to link more than the word "here". It's a good a11y practice to link phrases for when users are navigating by link in a screen reader.